docs/learn/advanced/18-syncing.md

```
@@ -0,0 +1,96 @@
+discovery_time = "15s" # time to spend discovering snapshots before picking one
+temp_dir = "" # directory for temporary state sync files
+chunk_request_timeout = "10s" # timeout for chunk requests
+chunk_fetchers = "4" # number of concurrent chunk fetchers
+```
+
+### Checking if sync is complete
+
+Query for the node status using the REST or GRPC API:  
+
+REST example:  
+```bash  
```

Markdownlint (0.37.0)

Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)